### PR TITLE
Update html.rst

### PR DESCRIPTION
**Pull request by icaroscherma**
Add a description to the function getCrumbList with an example.

### CODE REVIEW

File: en/views/helpers/html.rst [hunk 1]
```diff
@@ -805,6 +806,20 @@ You can also fetch the crumbs formatted inside an HTML list::
 
     echo $this->Html->getCrumbList();
 
+As options you can use regular Html parameter that fits in the ``<ul>``
```
Comment (dereuromark) [inline en/views/helpers/html.rst]: HTML

File: en/views/helpers/html.rst [hunk 2]
```diff
@@ -805,6 +806,20 @@ You can also fetch the crumbs formatted inside an HTML list::
...
+(Unordered List) such as ``class`` and for the specific options, you have:
+``separator`` (will be between the <li> elements), ``firstClass`` and ``lastClass``
```
Comment (dereuromark) [inline en/views/helpers/html.rst]: `<li>` should also be code fenced.

**Comment by icaroscherma** [comment]
I was checking the generated file and it is a bit odd. In the function getCrumbList, the arguments aren't being displayed properly (comma is before the []), check line 775 (there is normal, but the generated file is not. Also I think that I leaved a space before "like:" (line 812), leading to a unwanted indent. I will remove the space, but about the first "error", what should we do?

**Comment by cake17** [comment]
it's probably because of the `:` which should be `::` for the second error. I fixed it in 5acb367

**Comment by icaroscherma** [comment]
Just noticed that you changed while I was looking where can I send another pull request from the same point that I edited. Still getting used to GitHub website. I will see what I can help with the portuguese translation of the docs.

**Comment by antograssiot** [comment]
This line is very odd : 
http://book.cakephp.org/3.0/en/views/helpers/html.html#Cake\View\Helper\HtmlHelper::getCrumbList

``Cake\View\Helper\HtmlHelper::getCrumbList(array $options =, []$startText = false)``
don't know why the bracket are after the comma :disappointed:

**Pull request merged**